modules/gamelist-ui/src/lib/Button/Button.tsx

```diff
@@ -14,12 +14,14 @@ const buttonVariants = cva(
         secondary: 'bg-secondary text-secondary-foreground hover:bg-secondary/80',
         ghost: 'hover:bg-accent hover:text-accent-foreground',
         link: 'text-primary underline-offset-4 hover:underline',
+        loginSignUp: 'text-secondary bg-[#3db4f2] rounded-md mt-2.5 cursor-pointer font-semibold',
       },
       size: {
         default: 'h-10 px-4 py-2',
         sm: 'h-9 rounded-md px-3',
         lg: 'h-11 rounded-md px-8',
         icon: 'h-10 w-10',
+        loginSignUp: 'py-[10px] px-[30px]',
```

Comment:
Put something like: py-8 px-8 here, remove the hard coded px values

Reply:
@Yzma check this out:
[remove px values](https://github.com/tienviet10/gamelist-fe-v3/pull/16/commits/f42baa4274a47bc9ff2459f7387a844fd61f100c)